paapi.go

```diff
@@ -238,20 +242,34 @@ func (client *PAExternalAPIClient) GetResources(groupFilters []GroupFilter) (*Mo
 				if err != nil {
 					return nil, err
 				}
+				metricTitles := make(map[string]int)
 				for _, metric := range values.Infos {
-					newMetric := MonitoredValue{
-						GroupID:        group.ID,
-						ServerID:       server.ID,
-						MonitorValue:   metric.Status,
-						MonitorStatus:  metric.Status,
-						MonitorTitle:   metric.Title,
-						MonitorLastRun: metric.LastRun.Time,
+					// metric title is not unique
+					// discarding duplicate monitors
+					if _, titleExists := metricTitles[metric.Title]; titleExists {
+						metricTitles[metric.Title]++
+						log.Warnf("Duplicate monitor %s for server %s!", metric.Title, server.Name)
+					} else {
+						metricTitles[metric.Title] = 1
+
+						newMetric := MonitoredValue{
+							GroupID:        group.ID,
+							GroupName:      group.Name,
+							GroupPath:      group.Path,
+							ServerID:       server.ID,
+							ServerName:     server.Name,
+							MonitorValue:   metric.Status,
+							MonitorStatus:  metric.Status,
+							MonitorTitle:   metric.Title,
+							MonitorLastRun: metric.LastRun.Time,
+							MonitorID:      metric.ID,
+						}
+						metrics.Values = append(metrics.Values, newMetric)
 					}
-					metrics.Values = append(metrics.Values, newMetric)
 				}
 			}
 		} else {
-			log.Debugf("group named %s not found", filter)
+			log.Warnf("group named %s not found", filter)
```

Comment:
I would add precision here for users:
```suggestion
			log.Warnf("The configured group named %s was not found. It will be ignored.", filter)
```